README.md

```diff
@@ -87,6 +87,9 @@ The action can be configured with the following arguments:
   `true`. This is in an effort to reduce verbosity - if you want to have a
   comment per PR run, please ensure that you set this to `false`.
 
+- `comment-max-character` - (optional) Maximum number of characters of the Pulumi
```

Comment:
Curious what the value is of making this configurable?  Iiuc we have the maximum character limit because there is a maximum character limit for GitHub comments (which seems to be 65535 https://github.com/orgs/community/discussions/27190#discussioncomment-4803699).  

So at best we can allow someone to reduce the maximum character count, but we should error out if this setting is set above 64000.

Reply:
You are right, with the limit of GitHub comments in mind the only option would be to lower the comment length. Let me remove the configuration option again and put it as a const in the pr.ts file instead.